src/box/alter.cc

```diff
@@ -3897,7 +3897,7 @@ on_replace_dd_schema(struct trigger * /* trigger */, void *event)
 			return -1;
 		REPLICASET_UUID = uu;
 		box_broadcast_id();
-		say_info("cluster uuid %s", tt_uuid_str(&uu));
+		say_info("replicaset uuid %s", tt_uuid_str(&uu));
```

Comment:
What about `box.space._schema:get('cluster')`, `box.space._cluster`, `box.info.cluster`? Are there any plans to rename those as well? Otherwise, it looks inconsistent...

Reply:
Those can not be renamed so simple. Safely we can only rename texts like logs and not yet released features like `box.iproto.key.CLUSTER_UUID`. For the rest there is a backward-compatible plan, which I make live in the rest of #5029 (you can revisit the plan in the instance/replicaset/cluster names RFC).

I am trying to submit this patch asap because once `CLUSTER_UUID` is released soon, we won't be able to safely rename it either.